pkgs/applications/networking/cluster/nerdctl/default.nix

```
@@ -0,0 +1,30 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "nerdctl";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "AkihiroSuda";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "181qapqgp7zd0imk0zkn4wzpsw292ai2yz9pbiirpjcjx9h26w5h";
+  };
+
+  vendorSha256 = "0scywhllxk1m6456wggdmn7sgvy5x3gz2xnyfq9jnvvzap8byr2v";
+
+  buildFlagsArray = [
+    "-ldflags="
+    "-w"
+    "-s"
+    "-X github.com/AkihiroSuda/nerdctl/pkg/version.Version=v${version}"
+  ];
+
```

Is there a reason that you're copying the podman/cri-o wrapper rather than having a basic wrapper?

```suggestion

  postInstall = ''
      makeWrapper ${nerdctl-unwrapped}/bin/nerdctl $out/bin/nerdctl \
      --prefix PATH : ${lib.makeBinPath [ buildkit ]} \
      --prefix CNI_PATH : "${cni-plugins}/bin"
  '';
```

---

Well if you dont need networking or building containers then the unwrapped version would suffice.

I guess networking is pretty vital so i could just always wrap the cni plugins in, people could always override it to remove it.

I can either wrap buildkit in by defaut or leave users to install it seperately if they want

What would you recomend?

---

I think wrapping both by default is fine for now, we could switch to separate wrapper with an `unwrapped` version later if it's needed.

---

Ill swap it in a bit

---

Sorry for the delay. Fixed it.